packages/web-frontend/src/utils/getUniqueViewId.js

```diff
@@ -4,4 +4,10 @@
  */
 
 export const getUniqueViewId = ({ organisationUnitCode, dashboardGroupId, viewId }) =>
-  [organisationUnitCode, dashboardGroupId, viewId].join('_');
+  [organisationUnitCode, dashboardGroupId, viewId].join('___');
```

Comment:
The delimiter needed to be changed to something unique (because of the below function) and I used `___` as a placeholder. What do you think of it or what could be better? 
`THIS_IS_A_UNIQUE_DELIMITER_83647`? 
`|&|`?

Reply:
I think it's a great delimiter!